internal/minioclient/client.go

```diff
@@ -54,40 +56,48 @@ func NewMinio(conf Config) (Client, error) {
 }
 
 // MakeBucket makes a new bucket.
-func (client *Minio) MakeBucket(bucket, location string) error {
-	err := client.API.MakeBucket(bucket, location)
+func (client *Minio) MakeBucket(bucket, location string) (err error) {
+	defer mon.Task()(nil)(&err)
+
+	err = client.API.MakeBucket(bucket, location)
 	if err != nil {
 		return MinioError.Wrap(err)
 	}
 	return nil
 }
 
 // RemoveBucket removes a bucket.
-func (client *Minio) RemoveBucket(bucket string) error {
-	err := client.API.RemoveBucket(bucket)
+func (client *Minio) RemoveBucket(bucket string) (err error) {
+	defer mon.Task()(nil)(&err)
+
+	err = client.API.RemoveBucket(bucket)
 	if err != nil {
 		return MinioError.Wrap(err)
 	}
 	return nil
 }
 
 // ListBuckets lists all buckets.
-func (client *Minio) ListBuckets() ([]string, error) {
+func (client *Minio) ListBuckets() (names []string, err error) {
+	defer mon.Task()(nil)(&err)
+
 	buckets, err := client.API.ListBuckets()
 	if err != nil {
 		return nil, MinioError.Wrap(err)
 	}
 
-	names := []string{}
+	names = []string{}
```

Comment:
i think this can be removed, now that return variable `names []string` is defined

Reply:
done